About/about.html

```diff
@@ -0,0 +1,152 @@
+      <div class="aboutHero hero">
+        <p class="text-center bg-white col-md-6 mx-auto p-3">
+          The eye-opening e-comm whitepaper brings together the community of
+          experts and academics who compromise the creative affiliate network in
+          different countries every two years. <br />
+          This year we hope to expand our invitation list including
+          organizations and individuals
+        </p>
+        <p class="text-center pb-5 pt-3 col-md-6 mx-auto">
+          Kindly contact us via email for any further question about OXJAYENT
+          E-COMM WHITEPAPER 2022 <br />
+          support@oxjayent.com
+        </p>
+      </div>
+
+      <section class="aboutProgram mainProgram text-white">
+        <div
+          class="Program forum d-flex align-items-center d-md-flex justify-content-center gap-3"
+        >
+          <img src="../img/icons-3.svg" alt="" />
+          <div>
+            <h2>E_COMM the future of Marketing</h2>
+            <p>
+              It's imposible to overstate the importance of e-commerce in the
+              world of today. If you ever bought things and delivered to you
+              online, it is the best experience in shopping.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section id="event" class="bestEcom speakers">
+        <h2 class="text-center mt-4 mb-3">The Best E-comm Blueprint Ever</h2>
+        <div class="divider mb-4"></div>
+        <p class="text-center">
+          Take a look at the best recent e-comm whitepaper ever
+        </p>
+        <div class="d-md-flex gap-4 flex-wrap col-md-9 mx-auto">
+          <div class="twentyThirteen text-white col-md-5 mb-2">
+            <p class="text-center h1 pb-5">2018</p>
+            <p class="text-center h5">The Abuja Hub Ecom Submmit</p>
+          </div>
+
+          <div class="twentyThirteen text-white col-md-5 mb-2">
+            <p class="text-center h1 pb-5">2020</p>
+            <p class="text-center h5">The Abuja Hub Ecom Submmit</p>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer id="sponsors" class="mt-4">
+      <div class="footerIcons">
+        <h2 class="text-center">Sponsers</h2>
+        <div class="dividerFooter"></div>
+        <div class="icons">
+          <img class="icons11" src="../img/sponsor-1.png" alt="" />
+          <img class="icons1" src="../img/sponsor-2.png" alt="" />
+          <img class="icons1" src="../img/sponsor-3.png" alt="" />
+          <img class="icons1" src="../img/sponsor-4.png" alt="" />
+        </div>
+      </div>
+      <div
+        class="copyRight d-flex justify-content-beetween w-75 mx-auto align-items-center py-5"
+      >
+        <h2 class="p-2">
+          <a class="logo" href="../Home/index.html"
+            ><img src="../img/logo.jpeg" alt="logo"
+          /></a>
+        </h2>
+
+        <p class="p-2">
+          &copy; Project make by John, Omachi <br />
+          All right reserved
```

Comment:
- [ ] I think here also you need to make the font weight and font size simialar to the original design so kindly fix it to be the same since it is one of the requirements to make your website identical to the given design

- [ ] Kindly make the padding here smaller than it is in your design to match the design ✅💪🏻
- Your design
![foot](https://user-images.githubusercontent.com/58553711/181144394-9c5e7758-c01a-437a-9582-c2676b7c2fa0.PNG)

- Original design
![image](https://user-images.githubusercontent.com/58553711/181144463-0992fecd-89cd-455d-81f0-2e87e63a555c.png)

Reply:
footer readjusted